### ISSUE TITLE
Re-add all devices

### ISSUE THREAD

**Issue by HerrieMan72** [stas-demydiuk/domoticz-zigbee2mqtt-plugin#86]
Hi,
accidentally i added two zigbee2mqtt bridges in my domoticz hardware config via your plugin.
then I deleted one of them, but this also deleted all my zigbee connected devices from domoticz. I have in total around 50, so i don't fancy re-pairing all of them.
Now I would like to have the plugin re-create all my devices, but this doesn't seem to work.
I've tried removing all zigbee2mqtt bridges, restarting domoticz, adding again the zigbee2mqtt bridge, but nothign happens, none of my former zigbee devices are created :(
Can you please help if somehow I can force the plugin to re-create all my devices again ?

**Comment by HerrieMan72** [comment]
I checked with the debugging option enabled and then I get all sorts of 'adding device for..' messages in my logging, so it seems like the plugin is trying to add the devices but somehow these are not showing up in domoticz.
Help would be greatly appreciated.

**Comment by JanJaapKo** [comment]
Hi @HerrieMan72 

Normally when you execute an update to an installed instance of Zigbee2MQTT plugin it reads the list of devices that are in the Zigbee2MQTT broker (all devices that you can find in the config.yaml). Please note that devices in Domoticz are added but not set to used, so they should appear under the setup->devices tab, just not in the 'switches' tab.

**Comment by stas-demydiuk** [comment]
@HerrieMan72 as @JanJaapKo mentioned, plugin checks all available devices each time bridge goes online as well as adding or removing devices. Additionally you could force devices to sync by simply restarting the plugin.

**Comment by HerrieMan72** [comment]
Hi,
I still can't get this to work again :(
I've restarted the Pi where I have zigbee2mqtt running on
I've restarted the server where my domoticz is running on.
I've modified the list of paired zigbee devices to zigbee2mqtt.
I've removed and re-added the plugin to domoticz, with same name and also with different name.
I see in the domoticz logging correct messages from the plugin listing all the paired devices to zigbee2mqtt, so the plugin is receiving the correct data from zigbee2mqtt.
But none of these devices actually show up on my domoticz under the devices tab :((
It used to work before without problem, it stopped working after I accidentally added two instances of the plugin to domoticz and then deleted one. When I deleted one, all my zigbee devices where also deleted from domoticz and i'm unable since to get them back.
Any help in debugging this would be greatly appreciated.
